integrationtest/wildfly/pom.xml

```diff
@@ -173,6 +173,11 @@
             <artifactId>jboss-transaction-api_1.2_spec</artifactId>
             <scope>provided</scope>
         </dependency>
+        <dependency>
```

Comment:
Is this because of `@Generated` et al?

Reply:
Because of `@PostConstruct`. Seems it was moved out of the core libraries in JDK9, so we need a dependency.